### DISCUSSION TITLE
NDI on OBS 28?

### DISCUSSION THREAD

**Discussion by datlaunchystark** [obsproject/obs-studio#7072] [Development Questions] [answered]
In the release notes, it says "Added an option for low latency audio buffering to Settings -> Audio, primarily designed for outputs like Decklink and **NDI**"  Could this mean that there's plans that NDI will be integrated into OBS 28?  The current NDI plugin which hasn't been updated since 2020 no longer works in the 28 beta due to being a qt5 plugin however the release notes makes me think that there's plans to support it in this upcoming release.

**Comment by Fenrirthviti** [comment]
There are no plans at this time, due to the grey area licensing of the NDI libraries.

EDIT: However, the current maintainer is aware and is working towards a version compatible with OBS 28 by release.

**Reply by jbcurler2010** [reply]
This is spectacular news!

**Reply by D0jii** [reply]
Oof yea my whole dual pc streaming setup relies on NDI <3

**Reply by lennon2600** [reply]
there is still a way to do this with NDI and OBS for the video feed, if you're using NDI for the audio as well maybe have to find another work around (couple options)....just use the official NDI Webcam Input app, set webcam in that app as your gaming PC's NDI output, and add a webcam source in OBS (your NDI webcam input ), and voila, works just as well....

**Reply by kvnargueta** [reply]
Today I was updated to OBS 28 but the OBS-NDI Plugin not work, 50% of my Graphics Setup is by NDI. I had to go back to the previous version. I hope to you have the compatible update soon.

**Comment by ChicagoScanningMG** [comment]
Has the old version been updated for OBS 28 yet?

**Reply by RytoEX** [reply]
No. Please check for updates at the obs-ndi repo or on our [OBS Studio 28 Plugin Compatibility](https://obsproject.com/kb/obs-studio-28-plugin-compatibility) page.

**Comment by incenafilmes** [comment]
Version v28 of OBS has already been released and we have updated some machines here. We use the NDI OUTPUT for multiple independent outputs. It is a fundamental tool for us. Is there an update forecast?

**Reply by incenafilmes** [reply]
We used PTZ cameras with NDI HX protocol right in the scenes, and now we are prevented from using this feature as well.

**Comment by datlaunchystark** [comment]
If you really need NDI support for OBS 28 at this moment, there are some development builds which you could try at: https://github.com/Palakis/obs-ndi/actions
Keep in mind that these are still early builds so performance/stability may vary.  Also "Dedicated NDI Output" hasn't been implemented yet as far as I can tell.

**Reply by datlaunchystark** [reply]
So far no crashes yet with build 33 but still, mileage may vary.
https://github.com/Palakis/obs-ndi/actions/runs/2977380826

**Comment by jbcurler2010** [comment]
Thanks for posting. Started testing just now and no issues as of yet.

**Comment by aramg** [comment]
I've compiled and uploaded the current master branch against OBS 28, here
https://github.com/dev47apps/obs-ndi/releases/tag/4.9.2

Should be usable until the ongoing rewrite is complete.

**Reply by datlaunchystark** [reply]
That's awesome thanks so much!  So far runs great :)

EDIT: has distorted audio unfortunately but has "Dedicated NDI Output".

**Reply by adesuwa-osagie** [reply]
Encountered a buzzing sound (i.e. in the video) with this plugin.

**Reply by thesimpleone** [reply]
Thank you so much, this saved me this morning.

**Reply by aramg** [reply]
Will take a look at the audio problem, I only use video so didn't catch it

**Reply by ggracia** [reply]
Thank you! You're the best!

**Reply by aramg** [reply]
Version 4.9.2 has been posted on the dev47apps fork, please give it a try. 
I could reproduce the problem with stereo audio (not mono), and it seems to be working now with the fix.

**Reply by adesuwa-osagie** [reply]
> Version 4.9.2 has been posted on the dev47apps fork, please give it a try. I could reproduce the problem with stereo audio (not mono), and it seems to be working now with the fix.

Thank you for taking the time to address the audio! So far, so good - video quality is better then plugin version release back in 2020, and the audio is clear -  no more buzzing sound. 

Tested for about 2 minutes - Plan record for the next hour and then report back.

**Reply by adesuwa-osagie** [reply]
> > Version 4.9.2 has been posted on the dev47apps fork, please give it a try. I could reproduce the problem with stereo audio (not mono), and it seems to be working now with the fix.
> 
> Thank you for taking the time to address the audio! So far, so good - video quality is better then plugin version release back in 2020, and the audio is clear - no more buzzing sound.
> 
> Tested for about 2 minutes - Plan record for the next hour and then report back.

After about 1 1/2 of recording, no problems noted with the video or audio quality. 

Thank you, again, for taking the time to update the obs-ndi plugin, @aramg! 🙌🏾

**Comment by RytoEX** [comment, marked as answer]
The OBS Project does not develop or maintain the obs-ndi plugin. For status updates, please check our [OBS Studio 28 Plugin Compatibility page](https://obsproject.com/kb/obs-studio-28-plugin-compatibility) or [this pinned obs-ndi Issue](https://github.com/Palakis/obs-ndi/issues/745).
